src/app/components/pages/LogIn.tsx

```diff
@@ -286,11 +287,16 @@ export const LogIn = () => {
                                                 <RaspberryPiSignInButton/>
                                             </Col>
                                         </Row>}
-                                    <Row className="mb-3 justify-content-center">
+                                    <Row className={classNames("justify-content-center", siteSpecific("mb-2", "mb-3"))} >
                                         <Col sm={9}>
                                             <GoogleSignInButton/>
                                         </Col>
                                     </Row>
+                                    {isPhy && <Row className="mb-3 justify-content-center">
```

Comment:
Should this not be `mb-2` to mimic the Google one?
```suggestion
                                    {isPhy && <Row className="mb-2 justify-content-center">
```
I appreciate as it's the last one it might have needed additional spacing, but the container does that in this case.